src/Education/Resources.resx

```diff
@@ -130,7 +130,7 @@
     <value>[Education] Error displaying Rule Help Window: {0}</value>
   </data>
   <data name="RuleHelpToolWindowCaption" xml:space="preserve">
-    <value>Sonar Rule Help</value>
+    <value>SonarLint Rule Help</value>
```

Comment:
Changed to `SonarLint` for consistency with the other tool windows.
#3656 covers changing them all.